### PR TITLE
Move docs to the website

### PR DESCRIPTION
**Pull request by mshustov**
## Summary
This PR moves docs to the ClickHouse website to use the website as the main source of truth.
A counterpart work in clickhouse-docs repository: https://github.com/ClickHouse/clickhouse-docs/pull/792
Also, I moved some dev instructions to `CONTRIBUTING.md` guide.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+## Getting started
+ClickHouse-JDBC client is an open-source project, and we welcome any contributions from the community. Please share your ideas, contribute to the codebase, and help us maintain up-to-date documentation.
+
+### Create a fork of the repository and clone it
+```bash
+git clone https://github.com/[YOUR_USERNAME]/clickhouse-jdbc
```
Comment (zhicwu) [inline CONTRIBUTING.md:6]: Should be `clickhouse-java` right?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  clickhouseServer=x.x.x.x
+  # below properties are only useful for test containers
+  #clickhouseVersion=latest
+  #clickhouseTimezone=UTC
+  #clickhouseImage=clickhouse/clickhouse-server
+  #additionalPackages=
+  ```
+
+### Tooling
+We use [TestNG](http://org.testng/doc) as testing framework and for running ClickHouse Local instance [testcontainers](https://www.testcontainers.org/modules/databases/clickhouse/).
```
Comment (zhicwu) [inline CONTRIBUTING.md:101]: Should we format the doc so that there will be a blank line between header and paragraph?

**Review by zhicwu — CHANGES_REQUESTED** [review]
By the way, what's the PR for moving removed content to `clickhouse-docs`?

File: CONTRIBUTING.md (hunk 1 above)
Comment (mshustov) [reply CONTRIBUTING.md:6]: good catch

**Comment by mshustov** [comment]
> By the way, what's the PR for moving removed content to clickhouse-docs?

Update the PR title with a link to clickhouse-docs https://github.com/ClickHouse/clickhouse-docs/pull/792

**Comment by zhicwu** [comment]
Thanks @mshustov. The PR was for 0.4.0 but now it's 0.4.1 :) Will submit a PR in clickhouse-docs when I get home to fix that.

**Pull request merged**